.gitignore

```diff
@@ -46,6 +46,7 @@ compile_commands.json
 *.a
 
 # eclipse project files
+.idea/
```

Comment:
Recommend to use gitignore.io to general a full and managable intellij config

Reply:
Yes sir, I will

Reply:
Done!